manifests/t/TeamSonarr/Sonarr/3.0.9.1549/TeamSonarr.Sonarr.locale.en-US.yaml

```diff
@@ -0,0 +1,20 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: TeamSonarr.Sonarr
+PackageVersion: 3.0.9.1549
+PackageLocale: en-US
+Publisher: Team Sonarr
+PublisherUrl: https://sonarr.tv/
+PublisherSupportUrl: https://sonarr.tv/#support
+Author: Team Sonarr
+PackageName: Sonarr
+PackageUrl: https://sonarr.tv/
+License: GNU General Public License v3.0
+LicenseUrl: https://github.com/Sonarr/Sonarr/blob/develop/LICENSE.md
+Copyright: Creative Commons 3.0 License

```

Comment:
Creative commons is not a copyright. Please provide an actual copyright, or remove this field